sql/src/main/java/io/crate/expression/reference/sys/node/NodeStatsContextFieldResolver.java

```diff
@@ -118,7 +120,8 @@ public NodeStatsContextFieldResolver(ClusterService clusterService,
                                   ExtendedNodeInfo extendedNodeInfo,
                                   Supplier<ConnectionStats> psqlStats,
                                   Supplier<TransportAddress> boundPostgresAddress,
-                                  LongSupplier numOpenTransportConnections) {
+                                  LongSupplier numOpenTransportConnections,
+                                  Supplier<Long> clusterStateVersion) {
```

Comment:
Could we use a `LongSupplier` for the clusterStateVersion as well or is it really nullable?

Reply:
Ah good shout, it's not really nullable (initially thought it might be if there isn't any cluster state applied locally but that can't be the case)